### PR TITLE
New smashing website using Jekyll

### PR DESCRIPTION
**Pull request by dvinciguerra**

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      </div>
+    </nav>
+
+
+    <div class="col-md-offset-2 col-md-8 header-intro">
+      <div class="header-intro--text">
+        <h1>
+          <img src="{{ '/assets/images/smashing-home-text.svg' | prepend: site.url }}" alt="Smashing" width="100%"
+                                                                   />
+          <small>Sinatra based framework that lets you<br> build excellent dashboards</small>
```
Comment (terraboops) [inline _includes/header.html]: The `line-height` of this `small` is all mucked up on my browser, the text runs together. Could be fixed if we make this `small` into a `span` and change the `h1` to and `h2`.

**Review by terraboops — CHANGES_REQUESTED** [review]
There's one small issue, otherwise this looks amazing - thank you!

File: _includes/header.html (hunk 1 above)
Comment (dvinciguerra) [reply _includes/header.html]: Great... I will fix it!  Thanks. :)

Comment (dvinciguerra) [reply _includes/header.html]: Changed line-height and text element. 

Screenshot using Safari Browser and I tested at Firefox too.
image

**Comment by terraboops** [comment]
Awesome! Thanks!

**Comment by dvinciguerra** [comment]
Any time! :)

**Comment by RonanCodes** [comment]
Well done, looks amazing! 👏

**Pull request merged**